### PR TITLE
Added Gearman, RabbitMQ, Lodash and Underscore.js

### PR DESCRIPTION
**Pull request by yigitJF**
[Gearman](http://gearman.org)
[RabbitMQ](https://www.rabbitmq.com/)

### CODE REVIEW

**Comment by yigitJF** [comment]
[Lodash](https://lodash.com/)
[Underscore.js](https://underscorejs.org/)

File: src/technologies.js [hunk 1]
```diff
@@ -109,6 +111,7 @@ const technologies = [
   { name: "Python 3", released: new Date("2008-12-03"), icon: "python", link: "https://www.python.org/" },
   { name: "PyTorch", released: new Date("2016-08-26"), link: "https://pytorch.org/" },
   { name: "R", released: new Date("1993-08-01"), links: "https://www.r-project.org/" },
+  { name: "RabbitMQ", released: new Date("2007-02-08"), links: "https://www.rabbitmq.com/"},
```
Comment (disissid) [inline src/technologies.js]: change the property name to `link` (singular)

**Comment by yigitJF** [comment]
Updated!

File: src/technologies.js (hunk 1 above)
Comment (yigitJF) [reply src/technologies.js]: Done.

**Comment by jsrn** [comment]
Looks great, thank you!

**Pull request merged**